common/src/main/nl/jft/common/Identifiable.java

```diff
@@ -0,0 +1,17 @@
+package nl.jft.common;
+
+/**
+ * Represents an {@link Object} which can be identified by an id (see {@code getId()}).
+ *
+ * @author Lesley Vente
+ */
+public interface Identifiable {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Annotate the "Identifiable" interface with the @FunctionalInterface annotation (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1609)